### PR TITLE
add ppc64le / arm64 support

### LINKED ISSUE #36: Missing ppc64le package
Needs https://github.com/conda-forge/occt-feedstock/pull/59

### PR DESCRIPTION
**Pull request by matthiasdiener**
Fixes #36.

Needs https://github.com/conda-forge/vtk-feedstock/pull/16, https://github.com/conda-forge/occt-feedstock/pull/64, and https://github.com/conda-forge/fltk-feedstock/pull/6

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -1 +1,5 @@
+build_platform:
```
Comment (guyer) [inline conda-forge.yml:1]: Does this mean that it doesn't build on things that aren't linux?

Comment (matthiasdiener) [reply conda-forge.yml:1]: As far as I know, it builds on osx_64 too. We could add osx_arm64 in another PR.

Comment (guyer) [reply conda-forge.yml:1]: OK. I just wasn't sure whether `build_platform` was an inclusive list or an exclusive list.

Comment (guyer) [reply conda-forge.yml:1]: It looks like osx and win builds are queued, so that's evidently OK.

**Comment by guyer** [comment]
Linux failures appear to be because of dependencies on occt and vtk, so this PR will need to be rechecked once those other packages are successfully merged.

**Comment by isuruf** [comment]
Can you use native builds instead of cross compiling?

**Comment by isuruf** [comment]
Can you add `gmsh` to https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/arch_rebuild.txt so that the bot will automatically open PRs to all dependencies?

**Pull request merged**